pyramid_debugtoolbar/toolbar.py

```diff
@@ -156,6 +156,7 @@ def sget(opt, default=None):
         exc_history.eval_exc = intercept_exc == 'debug'
 
     def toolbar_tween(request):
+        request.pdbt_id = hexlify(id(request))
```

Comment:
I think this probably belongs lower. Probably just before where the toolbar is instantiated.